src/ir/comp.rs

```diff
@@ -668,30 +690,92 @@ impl CompFields {
         );
     }
 
-    fn deanonymize_fields(&mut self) {
+    fn deanonymize_fields(&mut self, ctx: &BindgenContext, methods: &[Method]) {
+        use std::collections::HashMap;
```

Comment:
Any particular reason that this isn't at the top level? If not, let's put it there.

Reply:
Sure